### PR TITLE
CNV-30685: RHEL subscription info

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.14+

Issue: [CNV-30685](https://issues.redhat.com//browse/CNV-30685)

Link to docs preview:
https://sabrinajess.github.io/previews/CNV30685/virt/getting_started/virt-web-console-overview.html#overview-settings-cluster_virt-web-console-overview
https://sabrinajess.github.io/previews/CNV30685/virt/virtual_machines/virt-create-vms.html#creating-vm-web_virt-create-vms
https://sabrinajess.github.io/previews/CNV30685/virt/vm_templates/virt-automatic-bootsource-updates.html#virt-about-auto-bootsource-updates_virt-automatic-bootsource-updates

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by dominikholler — APPROVED** [review]
looks good and correct from a technical point of view

File: modules/virt-about-auto-bootsource-updates.adoc [hunk 1]
```diff
@@ -9,8 +9,10 @@
...
 
 When the `enableCommonBootImageImport` feature gate is disabled, `DataSource` objects are reset so that they no longer point to the original PVCs. An administrator can manually provide a boot source by creating a new PVC for the `DataSource` object and populating the PVC with an operating system image.
 
-_Custom_ boot sources that are not provided by {VirtProductName} are not controlled by the feature gate. You must manage them individually by editing the `HyperConverged` custom resource (CR). You can also use this method to manage individual system-defined boot sources.
\ No newline at end of file
+_Custom_ boot sources that are not provided by {VirtProductName} are not controlled by the feature gate. You must manage them individually by editing the `HyperConverged` custom resource (CR). You can also use this method to manage individual system-defined boot sources.
+
+Cluster administrators can enable automatic subscription for {op-system-base-full} VirtualMachines in the {virtproductname} xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
```
Comment (mburke5678) [inline modules/virt-about-auto-bootsource-updates.adoc:18]: Should VirtualMachines should be two words, both lowercase? For what it's worth, I found this RH virualization doc: [Chapter 2. Installing Red Hat Enterprise Linux Virtual Machines](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/virtual_machine_management_guide/installing_red_hat_enterprise_linux_virtual_machines#doc-wrapper)

File: virt/virtual_machines/virt-create-vms.adoc [hunk 2]
```diff
@@ -28,6 +28,8 @@ You can expedite VM creation by using templates that have an available boot sour
 
 Templates without a boot source are labeled *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
+Cluster administrators can enable automatic subscription for {op-system-base-full} VirtualMachines in the {virtproductname} xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
```
Comment (mburke5678) [inline virt/virtual_machines/virt-create-vms.adoc:31]: Should VirtualMachines should be two words, both lowercase?

File: modules/virt-about-auto-bootsource-updates.adoc (hunk 1 above)
Comment (sabrinajess) [reply modules/virt-about-auto-bootsource-updates.adoc:18]: It sure should! Will fix.

File: virt/getting_started/virt-web-console-overview.adoc [hunk 3]
```diff
@@ -251,6 +251,11 @@ To store Red Hat templates in multiple projects, xref:../../virt/getting_started
 
 Preview features are disabled by default and must not be enabled in production environments.
 
+|*Automatic subscription of new RHEL VirtualMachines* section
+|Expand this section to enable automatic subscription for {op-system-base-full} VirtualMachines.
+
+Cluster administrator permissions are required to enable this feature.
```
Comment (apinnick) [inline virt/getting_started/virt-web-console-overview.adoc:257]: This is missing info. The cluster admin must provide an organization ID and activation code. It's not like any cluster admin can enable the feature.

File: virt/getting_started/virt-web-console-overview.adoc [hunk 4]
```diff
@@ -251,6 +251,11 @@ To store Red Hat templates in multiple projects, xref:../../virt/getting_started
 
 Preview features are disabled by default and must not be enabled in production environments.
 
+|*Automatic subscription of new RHEL VirtualMachines* section
+|Expand this section to enable automatic subscription for {op-system-base-full} VirtualMachines.
```
Comment (apinnick) [inline virt/getting_started/virt-web-console-overview.adoc:255]: "VirtualMachines" should be "virtual machines" to align with OCP guidelines.

**Pull request merged**